src/graphorm/index.ts

```diff
@@ -490,6 +492,24 @@ const obj = new GraphORM({
       title: createSmartTitleField({
         field: 'title',
       }),
+      translation: {
+        jsonType: true,
+        transform: (
+          translations: Partial<Record<ContentLanguage, PostTranslation>>,
+          ctx: Context,
+        ): Record<keyof PostTranslation, boolean> => {
+          const translation = ctx.contentLanguage
```

Comment:
can we early return `{}` if translation is not available, think it will be cleaner to not worry about optional in reduce?

Reply:
I just thought early return `{}` before reduce if `translations[ctx.contentLanguage]` is undefined

```js
const translation = translations[ctx.contentLanguage]

if (!translation) {
  return {}
}

// ... reduce
```

Reply:
because technically each field is not required so if its not there we don't have to return `false`

Reply:
but if you want to return `false` then I would prefer just leaving it as is

Reply:
I can do as you suggest. I see GraphQL converts it to what I need anyways :lolsob: